test/rubygems/test_gem_installer.rb

```diff
@@ -1422,11 +1422,7 @@ def test_find_lib_file_after_install
     write_file File.join(@tempdir, "extconf.rb") do |io|
       io.write <<-RUBY
         require "mkmf"
-
-        CONFIG['CC'] = '$(TOUCH) $@ ||'
-        CONFIG['LDSHARED'] = '$(TOUCH) $@ ||'
-        $ruby = '#{Gem.ruby}'
-
+        $LDFLAGS << " -L#{@current_dir}" # to link with libruby without needing it installed
```

Comment:
This would break mswin, you'll need to use `MakeMakefile#libpathflag`.

Reply:
This would break mswin, you'll need to use MakeMakefile#libpathflag

Reply:
Looks like that is used on the `$LIBPATH` array, so I changed these lines to append to that instead.